### PR TITLE
fix(editor): Fix cloud plan data loading on instance

### PR DESCRIPTION
**Pull request by MiloradFilipovic**
Moving cloud hooks and store initialization logic after users are authenticated. This will ensure user local account is available when their cloud plan data is being fetched.
This PR also adds the following error handling improvements:
- Added error handling to the same initialization logic
- Fixed empty `catch` clauses inside the cloud store which caused it to silently fail and complicated debugging of this bug

### CODE REVIEW

File: packages/editor-ui/src/init.ts [hunk 1]
```diff
@@ -63,5 +60,17 @@ export async function initializeAuthenticatedFeatures() {
...
+	if (settingsStore.isCloudDeployment) {
+		const results = await Promise.allSettled([cloudPlanStore.initialize(), initializeCloudHooks()]);
+		results.forEach((result, index) => {
+			if (result.status === 'rejected') {
+				console.error(
+					`Failed to initialize ${index === 0 ? 'cloud plan store' : 'cloud hooks'}:`,
+					result.reason,
+				);
+			}
+		});
```
Comment (alexgrozav) [inline packages/editor-ui/src/init.ts:72]: The `initializeCloudHooks` call does not require the user to be authenticated and should happen in the `initializeCore` block so we can actually run the `app.mounted` hook as well.

**Review by alexgrozav — CHANGES_REQUESTED** [review]
Looks good! One small change remaining.

File: packages/editor-ui/src/init.ts (hunk 1 above)
Comment (MiloradFilipovic) [reply packages/editor-ui/src/init.ts]: Ah, yes. Moved.

**Review by alexgrozav — APPROVED** [review]

**Review by alexgrozav — DISMISSED** [review]

**Review by OlegIvaniv — APPROVED** [review]
Looks good to me 👌 
Just a nitpick: PR description mentions:
> Using Promise.allSettled instead of Promise.all ... 

But that no longer seems to be the case. Could you update it, please?

**Comment by MiloradFilipovic** [comment]
> Looks good to me 👌 Just a nitpick: PR description mentions:
> 
> > Using Promise.allSettled instead of Promise.all ...
> 
> But that no longer seems to be the case. Could you update it, please?

Good eye 🕵️‍♂️

**Comment by janober** [comment]
Got released with [`n8n@1.18.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.18.1)

**Pull request merged**